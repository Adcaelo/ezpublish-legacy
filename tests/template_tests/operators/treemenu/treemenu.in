{var $a=treemenu(array('/'), 0)}
{$a[0]["id"]}
{$a[0]["url"]}
{$a[0]["url_alias"]}

