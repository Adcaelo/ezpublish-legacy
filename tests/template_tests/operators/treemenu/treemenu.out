69
/content/view/full/69/
/Products
